_shared/project/bin/make_db

```diff
@@ -0,0 +1,5 @@
+"""Initialize the dev environment's DB."""
+#!/usr/bin/env python3
+import sys
+from pyramid.paster import bootstrap
+bootstrap("conf/development.ini")
```

Comment:
The `make db` script as found in LMS, Checkmate and h's main branches (where it's called `initialize_db.py`, but here I'm following the nice `bin/make_*` naming convention that the cookiecutter uses).